src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/models/responses/UserResponse.kt

```diff
@@ -7,12 +7,22 @@ data class UserResponse(
   val id: String,
   val displayName: String,
   val password: String?,
-  val email: String
+  val email: String,
+  val libraries: List<LibraryResponse>?
 ) : Response() {
   constructor(id: String, displayName: String, email: String) : this(
     id = id,
     displayName = displayName,
     password = null,
-    email = email
+    email = email,
+    libraries = null
```

Comment:
So are we going to allow libraries to be null? Shouldn't we always initialize a user with one?

Reply:
In `UserService` when we create a User (method `create()`, the library gets initialized there. This constructor isn't currently in use, so I'll remove it.